types/bunyan-bugsnag/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for bunyan-bugsnag 1.0
+// Project: https://github.com/marnusw/bunyan-bugsnag
+// Definitions by: Pasi Eronen <https://github.com/pasieronen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as Logger from 'bunyan';
+
+interface StreamOptions {
```

Comment:
If you want consumers of your declaration file to be able to reference this interface, you should put it in the namespace you've defined.